doc/api/fs.md

```diff
@@ -1381,6 +1381,12 @@ the specified file descriptor. This means that no `'open'` event will be
 emitted. Note that `fd` should be blocking; non-blocking `fd`s should be passed
 to [`net.Socket`][].
 
+Also note that the blocking fd, if pointing to a character device (such as
```

Comment:
I may be wrong, but there are two confusing moments for me:
1. How `that the blocking fd...` correlates with the `creating a stream out of such a device can...`?
2. Singular vs plural: `a character device` vs `are end-less streams and do not produce`.

Reply:
"fd" should be in backticks, I think. (Can go either way, but for consistency with the previous paragraph, for example.)

You can probably drop `Also note that`.